tikv/tikv
# sst_importer: add wire extra bytes into the packed size

### What is changed and how it works?

Issue Number: Close #14313

What's Changed:
This PR added some extra bytes for each modification, so the `unpacked_size` would be nearer to the real raft entry size.


```commit-message
```


### Check List 

Tests 

- Manual test (add detailed scripts or steps below)

I have tested it in the internal cluster, if you are a PingCAP employee, you can check it at [here](https://tcms.pingcap.net/dashboard/executions/case/3878188). There aren't suitable screenshots to be published, sorry for that.

And I guess this can also be checked by any table with average row size less than 70B (see the comments), but for now I don't have suitable cluster for running such tests...

### Release note 

```release-note
None
```
(This is a fix over the master branch.)
